### PR TITLE
Provide updated content about the project aims & motivation

### PR DESCRIPTION
**Pull request by berrange**
This PR uses content that we collaboratively came up with during the f2f meetings.

### CODE REVIEW

**Comment by berrange** [comment]
Ping, for reviews

File: about.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+* This is a poor use of developer resources because multiple projects are
+  reinventing the same wheels.
+* This is a poor experience for cloud administrators as they have to manually
+  partition up their physical machines between the three separate applications,
+  and then manage three completely separate pieces of infrastructure.
+* This is a poor experience for tenant users because they have to learn three
+  completely different application APIs and frontends depending on which
+  particular type of workload they wish to run
+
```
Comment (fabiand) [inline about.md]: >While libvirt has been successful at providing a unified single-host centric mangement API for virtualization apps to build upon, nothing similar has  arisen to fill the gap for network level compute management to the same degree. 

Was dropped, it's okay to me. On the other hand we could have used it, or rather use it in the end of this document, to express the analogy of what libvirt is to the host, kubevirt is to the cluster. or something along the lines.

Comment (berrange) [inline about.md]: We've got a separate doc written by David which is covering the relationship to other technologies like ovirt/openstack/libvirt/etc. Something similar to this dropped paragraph would best live in that doc was my thought.

Comment (fabiand) [inline about.md]: Good idea.

**Pull request merged**